tests/MongoDB.Analyzer.Tests.Common.TestCases/Jira/VS78.cs

```diff
@@ -122,34 +122,35 @@ where u.LastName.Length < 10
                 select u.Name;
         }
 
-        [NoDiagnostics]
-        public void IQueryable_extensions_in_middle_and_prefix_should_be_ignored()
-        {
-            _ = GetMongoQueryable()
-                .Where(u => u.Name == "Bob" && u.Age > 16 && u.Age <= 21)
-                .Where(u => u.LastName.Length < 10)
-                .ApplyPagingIQueryable(1, 0)
-                .Where(u => u.LastName == "Smith");
-
-            _ = GetMongoQueryable()
-                .ApplyPagingIQueryable(1, 0)
-                .Where(u => u.Name == "Bob" && u.Age > 16 && u.Age <= 21)
-                .Where(u => u.LastName.Length < 10);
-
-            _ = GetMongoQueryable()
-                .ApplyPagingIQueryable(1, 0)
-                .Where(u => u.Name == "Bob" && u.Age > 16 && u.Age <= 21)
-                .Where(u => u.LastName.Length < 10);
-
-            _ = GetMongoQueryable()
-              .ApplyPaging(1, 0)
-              .ApplyPagingIQueryable(1, 0)
-              .Where(u => u.Name == "Bob" && u.Age > 16 && u.Age <= 21)
-              .Where(u => u.LastName.Length < 10);
-        }
-
-        [MQL("aggregate([{ \"$match\" : { \"Name\" : \"Bob\", \"Age\" : { \"$gt\" : 16, \"$lte\" : 21 } } }, { \"$match\" : { \"LastName\" : /^.{0,9}$/s } }])")]
-        [MQL("aggregate([{ \"$match\" : { \"Name\" : \"Bob\", \"Age\" : { \"$gt\" : 16, \"$lte\" : 21 } } }, { \"$match\" : { \"LastName\" : /^.{0,9}$/s } }])")]
+        // VS-163
```

Comment:
Should either remote this code or put some commentary around it as to why it's here but commented out.